hnn_core/network.py

```diff
@@ -750,6 +753,40 @@ def _instantiate_drives(self, n_trials=1):
                 self.external_drives[
                     drive['name']]['events'].append(event_times)
 
+    def connect_cells(self, src_gids, target_gids, weight, receptor, location,
+                      delay, space_constant):
+        """Pass connectivity matrix defining cell-cell connections.
+
+        Parameters
+        ----------
+        src_gids: list of int
+            List of gid sources where axon originates.
+        target_gids: list of int
+            List of gid targets where axon terminates.
+        weight: np.array (n_src_gids, n_target_gids) of float
+            Array defining synaptic weights.
```

Comment:
So the weight and the space constant is combined to get the final cell-cell weight, right? Should the space_constant be set to 0 (or whatever makes it turn off) by default? I'm worried there will be weird interactions that the user won't understand

Reply:
Looking at the documentation of NEURON, threshold should be removed as well (set to a fixed value). I think it actually corresponds to the presynaptic threshold, in which case there should absolutely not be axons with different AP thresholds on the same cell. Still need to double check that though.

Reply:
I just saw this comment thread. Also see my [comment](https://github.com/jonescompneurolab/hnn-core/pull/276#discussion_r597160625) here.

Reply:
Maybe this is what you were trying to say @ntolley, but just to clarify, "threshold" is the membrane potential at which an action potential results in synaptic relay (as opposed to the threshold at which an AP occurs within a cell) right?

Reply:
That is my understanding since it seems that different connections can have different thresholds.